src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]"); // mark done task with X
```

Comment:
I feel that the comments should be placed on a seperate line relative to the code